llm/ext_server/server.cpp

```diff
@@ -48,11 +48,13 @@
 #include <chrono>
 #include <condition_variable>
 #include <atomic>
+#include <algorithm>
 #include <signal.h>
 
 using json = nlohmann::json;
 
-struct server_params {
+struct server_params
```

Comment:
nit - thanks for running a formatter on the file, but maybe next time split the purely formatting changes into a discrete commit so the substantive changes are a little easier to review in isolation.